cli/multisig.go

```diff
@@ -658,7 +662,7 @@ var msigCancelCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() < 2 {
+		if cctx.NArg() != 2 {
```

Comment:
and again, there's 2 blocks below this that handle other cases up to 6 arguments, let's leave it alone